python/lsst/afw/table/ampInfo.cc

```diff
@@ -34,16 +36,112 @@ using namespace pybind11::literals;
 namespace lsst {
 namespace afw {
 namespace table {
+namespace {
+
+using PyAmpInfoRecord = py::class_<AmpInfoRecord, std::shared_ptr<AmpInfoRecord>, BaseRecord>;
+using PyAmpInfoTable = py::class_<AmpInfoTable, std::shared_ptr<AmpInfoTable>, BaseTable>;
+using PyAmpInfoCatalog =  py::class_<CatalogT<AmpInfoRecord>, std::shared_ptr<AmpInfoCatalog>>;
+
+void declareAmpInfoRecord(PyAmpInfoRecord & cls) {
+    table::pybind11::addCastFrom<BaseRecord, AmpInfoRecord>(cls);
+
+    cls.def("getName", &AmpInfoRecord::getName);
+    cls.def("setName", &AmpInfoRecord::setName, "name"_a,
+                         "Set name of amplifier location in camera");
+    cls.def("getBBox", &AmpInfoRecord::getBBox);
+    cls.def("setBBox", &AmpInfoRecord::setBBox, "bbox"_a);
+    cls.def("getGain", &AmpInfoRecord::getGain);
+    cls.def("setGain", &AmpInfoRecord::setGain, "gain"_a, "Set amplifier gain in e-/ADU");
```

Comment:
I'm afraid I don't understand what you are suggesting. Are you just commenting on the string used to describe the units? If so, what do you suggest? We can't change the actual units, of course.